src/chronobike.h

```diff
@@ -37,20 +36,16 @@ class chronobike : public bike {
     Q_OBJECT
   public:
     chronobike(bool noWriteResistance, bool noHeartService);
-    bool connected();
-
-    void *VirtualBike();
-    void *VirtualDevice();
+    bool connected() override;
 
   private:
     // void writeCharacteristic(uint8_t *data, uint8_t data_len, QString info, bool disable_log = false, // Unused
     //                         bool wait_for_response = false);
     void startDiscover();
-    uint16_t watts();
+    uint16_t watts() override;
```

Comment:
It's not mandatory, and it's not necessary for this PR, I'll remove it. 

The keyword is relatively recent development in the C++ language, used to indicate if a method is overriding a virtual method from a superclass. I believe it is preferable to use it because it distinguishes a method that is intended to override something from a method put in without that intent - you can't use the override keyword where it's not overriding something.

Reply:
> 

hah ok! maybe we can do a separate PR for this?